migrations/20241120002050-create-usages-index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const indexName = 'usage_user_type_period_index';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(
+      `CREATE INDEX CONCURRENTLY ${indexName} ON usages (user_id, type, period)`,
+    );
+  },
+
+  async down(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(
+      `DROP INDEX CONCURRENTLY ${indexName}`,
+    );
+  },
```

Comment:
This is just a tentative change, we might not need it since it is not really useful for calculating files with changed size